### DISCUSSION TITLE
prettier settings

### DISCUSSION THREAD

**Discussion by eskan** [meteor/meteor#11949] [Feature Requests]
the prettier config seems to be based on @quave/eslint-config-quave/prettier.config but when i "prettier" source code almost all lines are rewrite.
Do i miss something ?

**Comment by StorytellerCZ** [comment]
I think that sound right, as the prettier/eslint settings have changed over the time.

**Reply by eskan** [reply]
Thanks. do you have any instructions, i guess we need to keep current "config" for file ?
Do you plan to reformat sources in a global batch ? I'm interesting to know how you plan to handle this if you do... or just let all as is :D which is my personal preferred method ever
